cms/api.py

```diff
@@ -321,7 +322,9 @@ def create_page_content(language, title, page, menu_title=None, slug=None,
         language=language,
     )
 
-    page_content = PageContent.objects.create(
+    # E.g., djangocms-versioning needs an User object to be passed when creating a versioned Object
+    user = getattr(_thread_locals, "user", "unknown user")
```

Comment:
I'm not sure a default should be returned here, it needs to be an actual object not a string. If there's no user found it should error.

Reply:
@Aiky30 So far, the requirement for a user object was only introduced by versioning. While I'd second the same requirement for the core it breaks the entire test suite: 573 out of 1025 tests do not provide a user object. Once versioning is active versioning will raise an error if there is no User object. Hence the default leads to the same behavior as with 4.0.x.

Reply:
Ok.